docs/user/tutorial/30-structure-content-modes.rst

```diff
@@ -92,7 +94,10 @@ edit icon.
 
 You can switch back to content mode to see the effect of your changes.
 
-.. todo:: screenshot of the page showing the map
+.. todo:: Daniele please check screenshot of the page showing the map
+
+.. image:: /user/tutorial/images/page-with-google-map.png
+   :alt: the structure board with two plugins
```

Comment:
the alt text is incorrect here, should be "the Google Maps plugins shows the workshop location" or something.